Services/MovementService.cs

```diff
@@ -0,0 +1,70 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Zenny_Api.Data;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Zenny_Api.Models;
+
+namespace Zenny_Api.Services;
+
+public class MovementService
+{
+    private readonly MovementDbContext _context;
+
+    public MovementService(MovementDbContext context)
+    {
+        _context = context;
+    }
+
+    // get all the movements for the current month
+    public async Task<IEnumerable<Movement>> GetMovementsAsync()
```

Comment:
personally i will prefer a single function, but this is allso an option of single-proposit